README.md

```diff
@@ -21,6 +21,19 @@ To run the project you may specify a `.toml` configuration file, if none is spec
 ```
 A sample configuration file can be found at `config.toml.template`
 
+### Meta Bucket
```

Comment:
Configuration or System bucket seems to be more appropriate as we are not actually storing any metadata about data ingested by minsql

Reply:
We'll wait to discuss with @abperiasamy